src/js/components/Paragraph/StyledParagraph.js

```diff
@@ -43,13 +43,13 @@ const textAlignStyle = css`
   text-align: ${props => TEXT_ALIGN_MAP[props.textAlign]};
 `;
 
-const StyledParagraph = styled.p`
+const StyledParagraphBase = styled.p`
```

Comment:
yeah I like it here. so that the exported thing is always StyledParagraph

Reply:
I've updated this to avoid the `Base` suffix entirely.